satisfactory/egg-pterodactyl-satisfactory.json

```diff
@@ -4,27 +4,27 @@
         "version": "PTDL_v2",
         "update_url": null
     },
-    "exported_at": "2024-09-10T11:11:13-07:00",
+    "exported_at": "2024-09-19T23:39:05-04:00",
     "name": "Satisfactory",
-    "author": "red_thirten@yahoo.com",
+    "author": "vapokrocks@gmail.com",
     "description": "Satisfactory is a first-person open-world factory building game with a dash of exploration and combat. Play alone or with friends, explore an alien planet, create multi-story factories, and enter conveyor belt heaven!",
     "features": [
         "steam_disk_space"
     ],
     "docker_images": {
-        "ghcr.io\/parkervcp\/steamcmd:debian": "ghcr.io\/parkervcp\/steamcmd:debian"
+        "ghcr.io\/parkervcp\/games:source": "ghcr.io\/parkervcp\/games:source"
     },
     "file_denylist": [],
-    "startup": ".\/Engine\/Binaries\/Linux\/*-Linux-Shipping FactoryGame -Port={{SERVER_PORT}}",
+    "startup": ".\/FactoryServer.sh ?listen -port={{SERVER_PORT}} $(if {{DISABLE_SEASONAL}}; then echo \"-DisableSeasonalEvents\"; fi)",
```

Comment:
Startscripts are not allowed

Reply:
I mean it's the script that Coffee Stain has provided, which sets some environment variables.

Reply:
there is no ENV variable:

```
#!/bin/sh
UE_TRUE_SCRIPT_NAME=$(echo \"$0\" | xargs readlink -f)
UE_PROJECT_ROOT=$(dirname "$UE_TRUE_SCRIPT_NAME")
chmod +x "$UE_PROJECT_ROOT/Engine/Binaries/Linux/FactoryServer-Linux-Shipping"
> "$UE_PROJECT_ROOT/Engine/Binaries/Linux/FactoryServer-Linux-Shipping" FactoryGame "$@"
```